sympy/physics/units/tests/test_util.py

```diff
@@ -128,7 +128,3 @@ def test_eval_simplify():
     assert ((km/cm).simplify()) == 100000
     assert ((10*x*K*km**2/m/cm).simplify()) == 1000000000*x*kelvin
     assert ((cm/km/m).simplify()) == 1/(100*kilometer)
```

Comment:
`assert ((cm/km/m).simplify()) == 1/(100*kilometer)`
Why not use SI base-units as a standard, i.e. `1/(100000*meter)`? It was mentioned in the discussion that a random unit is picked, in this case kilometer, but I think the SI system provides an unambiguous reference that we should use.

Reply:
I believe that the simplification should actually choose the unit that gives the coefficient that is closest to one, so that the expression look simpler. If you want to convert to meters, there is the `convert_to` function. You can call `convert_to(expr, (<list of base units of SI>))`